run/rails/Dockerfile

```diff
@@ -7,23 +7,38 @@
 FROM ruby:3.2-buster
```

Comment:
question: if there's an issue with buster, why not update the base image to bookworm? Does that version have the same issue?